README.md

```diff
@@ -52,7 +52,8 @@ var TouchID = require('react-native-touch-id');
 
 var YourComponent = React.createClass({
   _pressHandler() {
-    TouchID.authenticate()
+  var reasonText = "Your reason string for the TouchID dialogue"
```

Comment:
This is partially my fault for not setting up an eslint config, but can you replace all double quotes in the PR with single quotes so that the project can have a consistent style.

I'll add an eslint config this week.